### PR TITLE
Xrole: Create reusable organisation card component

### PR DESCRIPTION
**Pull request by XROLE**
#### What does this PR achieve?
- [x] create reusable card component for displaying organisations
- [x] create organisational component story
- [x] add global style in storybook
- [x]  add dark_gray color theme in color library

#### Any background context?
- N/A

#### Manual Testing Steps
- run `npm run storybook` to spin up storybook server
- you should see organisation card component on the browser opened by storybook
- scroll to the buttom of  storybook, click on the `knob` tab and play around with it by passing different properties value

#### Screenshots:
Screenshot 2020-01-10 at 23 37 01

#### Relevant Trello Tickets
- [#47](https://trello.com/c/bmi352On/47-d-1-create-organisation-card-reusable-component)

### CODE REVIEW

File: src/components/ui/OrganisationCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { DARK_GRAY } from 'settings/__color';
+import styled from '@emotion/styled';
+
+export interface OrganisationCardProps {
+  img: string;
+  name: string;
+  desc: string;
```
Comment (chidioguejiofor) [inline src/components/ui/OrganisationCard/index.tsx]: I think you should change `desc` to role

Comment (chidioguejiofor) [reply src/components/ui/OrganisationCard/index.tsx]: That part of the card shows the role of a user in the organisation

**Review by chidioguejiofor — CHANGES_REQUESTED** [review]
Nice completing this one. I see that your card looks like the mock when I run storybook but doesn't look the same on mobile. 

I think you should modify your CSS and use chrome dev tools to test responsiveness  I have attached some images of how the card looks on mobile currently. 
Screenshot 2020-01-12 at 4 50 12 PM
Screenshot 2020-01-12 at 4 50 03 PM
Screenshot 2020-01-12 at 5 00 25 PM
Screenshot 2020-01-12 at 5 00 17 PM
Screenshot 2020-01-12 at 4 59 57 PM
Screenshot 2020-01-12 at 4 59 51 PM

File: src/components/ui/OrganisationCard/index.tsx (hunk 1 above)
Comment (XROLE) [reply src/components/ui/OrganisationCard/index.tsx]: > I think you should change `desc` to role

Ok, implement the feedback asap. Thank you

File: src/components/ui/OrganisationCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+  p {
+    color: ${DARK_GRAY};
+  }
+  @media (max-width: ${__layouts.xLg}) {
+    height: 16vh;
+  }
+
+  @media (max-width: ${__layouts.md}) {
+    height: 25vh;
```
Comment (chidioguejiofor) [inline src/components/ui/OrganisationCard/index.tsx]: This seems to cause some problems. Once you remove this the component is perfect on all phones


Screenshot 2020-01-13 at 9 55 37 PM
Screenshot 2020-01-13 at 9 55 25 PM

**Review by chidioguejiofor — CHANGES_REQUESTED** [review]
Nice job! Almost there

File: src/settings/__color.ts [hunk 3]
```diff
@@ -4,5 +4,6 @@ export const BRAND_WHITE = '#FFF';
 export const BLACK = '#494747';
 export const GRAY = '#888';
 export const LIGHT_GRAY = '#aaa';
+export const DARK_GRAY = '#7A7D7F';
```
Comment (chidioguejiofor) [inline src/settings/__color.ts:7]: This is nice

Comment (XROLE) [reply src/settings/__color.ts:7]: Thanks

File: src/components/ui/OrganisationCard/index.tsx (hunk 2 above)
Comment (XROLE) [reply src/components/ui/OrganisationCard/index.tsx]: Oh I see. Thank you. Implementing the feedback asap

**Pull request merged**